lib/scripts/scheduler.rb

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/ruby
```

Comment:
@Jullianos pls move this to a rake task.
Ex: https://stackoverflow.com/questions/28110384/how-to-parse-rake-arguments-with-optionparser

Reply:
I've created a scheduler.rake task, and this is the way you call the rake command:
log_ids: `> rake schedule_logs -- '--ids=1,24,55'`
time_frame: `> rake schedule_logs -- '--from=02-02-2022' '--to=03-03-2022'`